projects/kit/components/input-number/input-number.directive.ts

```diff
@@ -27,6 +27,6 @@ export class TuiInputNumberDirective
         }
 
         this.host.nativeFocusableElement.maxLength = this.host.calculatedMaxLength;
-        this.host.nativeFocusableElement.inputMode = 'decimal';
```

Comment:
Does it toggle `inputMode` for all cases ?

Maybe only ?
`this.isIOS && this.isNegativeAllowed ? 'decimal' : 'text'`

Reply:
https://github.com/Tinkoff/taiga-ui/blob/ca8a441f546f3c758132671e79963ff5faefb2cb/projects/kit/components/input-number/input-number.component.ts#L121-L128

Reply:
right now `get inputMode` doesn't usage anywhere

Reply:
after

![image](https://user-images.githubusercontent.com/12021443/225934027-e7dcf15b-2721-422e-a362-b136165bb641.png)

Reply:
it will be perfect to keep the same condition: show `text` keyboard when minus is required, otherwise show cool `decimal`-keyboard

Reply:
@splincode ```this.host.nativeFocusableElement.inputMode = this.host.inputMode;```

host is `TuiInputNumberComponent` where `inputMode` getter is  https://github.com/Tinkoff/taiga-ui/blob/ca8a441f546f3c758132671e79963ff5faefb2cb/projects/kit/components/input-number/input-number.component.ts#L121-L128